docs/gettingstarted/buildspecs_interface.rst

```diff
@@ -421,12 +421,12 @@ Edit buildspecs ``buildtest buildspec edit-file``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The ``buildtest buildspec edit-file`` command can be used to edit buildspec based on filename as pose to testname.
-This command works similar to ``buildtest buildspec edit`` where each file is open in editor and validated upon completion.
-You can use this command to create new buildspec whereas ``buildtest buildspec edit`` only works on existing buildspecs loaded
+This command works similar to ``buildtest buildspec edit-test`` where each file is open in editor and validated upon completion.
```

Comment:
I think you mean `buildtest buildspec edit-file` here since this is section on editing by buildspecs. The section above is for `edit-test`